content/applications/marketing/email_marketing.rst

```diff
@@ -1,440 +1,792 @@
+represents emojis that can be added to the :guilabel:`Subject` field. Clicking that icon reveals a
+pop-up menu of emojis that can be used.
 
-Next, there is the option to enter some :guilabel:`Preview Text`. This text is a catchy preview
-sentence that encourages recipients to open the message. In most inboxes, this is displayed next to
-the subject.
+Beside the :guilabel:`(smiley face with a plus sign)` icon at the end of the :guilabel:`Subject`
+field is an empty :guilabel:`(star)` icon. When clicked, the :guilabel:`(star)` icon turns gold, and
+the email is saved as a template in the :guilabel:`Mail Body` tab, which can be used again in the
+future.
 
-.. tip::
-   Keep the :guilabel:`Preview Text` empty to show the first characters of the email content,
-   instead.
+.. _email_marketing/recipients:
 
 Recipients
 ----------
 
-After that, it's time to choose the recipients of this email, which can be completed in the
-:guilabel:`Recipients` field.
+Beneath the :guilabel:`Subject` field on the email form is the :guilabel:`Recipients` field. In this
+field, select the recipients of the email. By default, the :guilabel:`Mailing List` option is
+selected, but clicking the field reveals a drop-down menu of other recipient options.
 
-.. image:: email_marketing/recipients-dropdown.png
-   :align: center
-   :alt: View of recipients drop-down menu in the Odoo Email Marketing application.
+With the default :guilabel:`Mailing List` option selected, a specific mailing list **must** be
+chosen from the adjacent :guilabel:`Select mailing lists` field drop-down menu.
 
-The default option is :guilabel:`Mailing List`. If :guilabel:`Mailing List` option is selected, a
-specific :guilabel:`Mailing List` needs to be chosen from the adjacent :guilabel:`Select mailing
-lists` field drop-down menu.
+.. tip::
+   More than one mailing list can be chosen from the :guilabel:`Select mailing lists` field.
 
-Then, Odoo will only send this email to contacts on that specific mailing list.
+Odoo then sends the email to contacts on that specific mailing list(s).
 
 .. seealso::
-   :doc:`/applications/marketing/email_marketing/mailing_lists`
+   :doc:`email_marketing/mailing_lists`
 
 When the :guilabel:`Recipients` field is clicked, a drop-down menu of other options is revealed.
 Each option provides different ways Odoo can create a target audience for the email.
 
+.. image:: email_marketing/recipients-dropdown.png
+   :align: center
+   :alt: View of recipients drop-down menu in the Odoo Email Marketing application.
+
 Those options (excluding the default :guilabel:`Mailing List`) provide the option to create a more
-specified recipient filter, in an equation-like format.
+specified recipient filter, in an equation-like format, which appears beneath the
+:guilabel:`Recipients` field.
 
-The :guilabel:`Recipients` field options are as follows:
+The :guilabel:`Recipients` field options, other than the default :guilabel:`Mailing List` option,
+are as follows:
```

Comment:
The descriptions for these list items are a borderline redundant. Would expand them so they're more specific to the models they're attached to if you can. 

For example: 
- **Lead/Opportunity** ties specifically to records in the _CRM_ application which opens up a number of opportunities to influence sales or purchase decisions (like what we do internally for lost lead reactivation drip campaigns).
- Event Registration ties specifically to the _Events_ app and provides opportunities to interact with event registrants to communicate important information about the event(s) or nurture other valuable actions such as post-event surveys or purchases, etc.

What is the difference between **Contact** and **Mailing Contact** ?